src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/shared/resources/queue/dataTask/dataTaskQueueSagas.ts

```diff
@@ -25,11 +25,8 @@ export function* startInitialDataTaskQueue(): SagaIterator {
 }
 
 export function* watchStartInitialDataTaskQueueSaga(): SagaIterator {
-  yield all([
-    take(START_INITIAL_DATA_TASK_QUEUE),
-    take(FETCH_APPLICATION_METADATA_FULFILLED),
-    take(GET_INSTANCEDATA_FULFILLED),
-  ]);
-
-  yield call(startInitialDataTaskQueue);
+  while (true) {
```

Comment:
Could we replace this `while(true)` loop with a `takeEvery` effect on the `START_INITIAL_DATA_TASK_QUEUE` action?

Reply:
Hmm good point! Will test to make sure it works as expected, but I assume it will :)